Automattic/wp-calypso
# Update Newspack to 1.13.1  - adds alignment options to post carousel block

This updates the version of Newspack Blocks from 1.13 to the latest release 1.13.1

### Changes proposed in this Pull Request
Update newspack version

### Testing instructions

- Check out PR
- Sync Newspack blocks in editor toolkit root with  `yarn run sync:newspack-blocks --release=v1.13.1`
- Run editor toolkit in local dev env and make sure post carousel block has alignment toolbar options

<img width="334" alt="Screen Shot 2020-09-30 at 11 41 44 AM" src="https://user-images.githubusercontent.com/3629020/94623983-f8e4be80-0311-11eb-9e22-1aac5c968730.png">
